azure-functions-maven-plugin/src/main/java/com/microsoft/azure/maven/function/DeployMojo.java

```diff
@@ -76,22 +76,21 @@
         " meet the requirement of Azure Functions, set it to Java 8.";
     public static final String UNKNOWN_DEPLOYMENT_TYPE = "The value of <deploymentType> is unknown, supported values are: " +
             "ftp, zip, msdeploy, run_from_blob and run_from_zip.";
-
     public static final String APPINSIGHTS_INSTRUMENTATION_KEY = "APPINSIGHTS_INSTRUMENTATIONKEY";
-    public static final String FAILED_TO_GET_APPLICATION_INSIGHT = "Application insight %s was not found, " +
+    public static final String APPLICATION_INSIGHTS_NOT_SUPPORTED = "Application Insights feature was not supported with" +
+            " current authentication, skip related steps.";
+    public static final String APPLICATION_INSIGHTS_CONFIGURATION_CONFLICT = "Can not set <appInsightsKey> or " +
```

Comment:
private static final?

Reply:
Contradicting configuration for application insights, specify 'appInsightsKey' or 'appInsightsInstance' if you want to enable it, and specify 'disableAppInsights=true' if you want to disable it.